python/paddle/distributed/auto_parallel/reshard.py

```diff
@@ -26,6 +26,9 @@
 from .dist_attribute import OperatorDistributedAttribute, TensorDistributedAttribute
 from .process_group import new_process_group, ProcessGroup, _g_process_group_map
 
+# NOTE: If op in SPECIAL_OPS, it will not be resharded. 
+SPECIAL_OPS = ['check_finite_and_unscale', 'update_loss_scaling']
```

Comment:
The global variable should use `_g_xxxx.` Please rename `SPECIAL_OPS `to `_g_special_ops`.

Reply:
done.